src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -105,6 +105,10 @@ class ContactMethodDetailsPage extends Component {
         };
     }
 
+    componentDidMount() {
+        User.resetContactMethodValidateCodeSentState(this.getContactMethod());
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/24521, this has caused a small regression 
Full details can be found [here](https://github.com/Expensify/App/issues/24521#issuecomment-1676549762), but in short we didn't check if the contact method exists before calling `resetContactMethodValidateCodeSentState`.
That meant you could edit URL to contain an invalid contact method and still proceed with the validation flow instead of being presented with the `Not Found` page